lib/feature_management.rb

```diff
@@ -70,7 +74,7 @@ def self.current_env_allowed_to_see_usps_code?
   end
 
   def self.fake_banner_mode?
-    Rails.env.production? && Figaro.env.domain_name != 'secure.login.gov'
+    Rails.env.production? && ENVS_DO_NOT_DISPLAY_FAKE_BANNER.exclude?(Figaro.env.domain_name)
```

Comment:
WDYT of checking `LoginGov::Hostdata.env` instead? That way it would be `staging` or `prod` and maybe easier to follow than domain name?

Reply:
Good point, however, in this case I think we are more concerned about configuration in upper/lower AWS environment. Stating identitysandbox.gov verses login.gov tells us exactly which AWS environment we are targeting.

Reply:
Ok! We have the `LoginGov::Hostdata.domain` for that, which returns `login.gov` in staging (I just verified now)

```
# int
LoginGov::Hostdata.domain
# staging
LoginGov::Hostdata.domain
```

and I think that would be cleaner than checking the specific IDP domain name, because there are just the two to check

Reply:
Ah, OK. I will enter a task/chore to convert all the constants in feature_management.rb for consistency.